_posts/release/2018-03-06-accumulo-1.9.0.md

```diff
@@ -22,15 +22,47 @@ Below are resources for this release:
 
 In [ACCUMULO-4611], public API in ClientConfiguration using commons config types was deprecated to
 better support Hadoop 3 in the future. New methods were added to replace these methods which cause
-this release to be a 1.9.0 release.
+this release to be a 1.9.0 release.  These changes allow removal of commons config from Accumulo's 
+API in 2.0.0.  If using ClientConfiguration, then switching from existing constructors to the new
+static methods [create()], [fromFile()], or [fromMap()] will ensure your code works in 2.0.0.
 
 ### Performance Improvements
 
-Several peformance improvements were added in [#379], [ACCUMULO-4778], [ACCUMULO-4779] and
-[ACCUMULO-4781].
+Accumulo was profiled while running lots of concurrent small scans. During this excercise these
+performance bugs were found and fixed: [#378], [ACCUMULO-4778], [ACCUMULO-4779], [ACCUMULO-4781],
+[ACCUMULO-4782], [ACCUMULO-4788], [ACCUMULO-4789], [ACCUMULO-4790], [ACCUMULO-4797], 
+[ACCUMULO-4798], [ACCUMULO-4799], [ACCUMULO-4800], [ACCUMULO-4801], [ACCUMULO-4805], and [ACCUMULO-4809]  
+
+### Fixed upgrade process to set version on all volumes
+
+During upgrades, only one volume in a multiple HDFS volume was updated with the correct version. This would cause all tablet servers
+to complain and ultimately fail. [ACCUMULO-4686] fixes this by setting the version on all volumes.
+
+### Updated Accumulo to work with new releases of Guava
+
+In [ACCUMULO-4702], dependencies on Beta-annotated Guava classes and methods were removed. While Accumulo still
+includes Guava 14 in its tarball, it will work with newer versions of Guava in client code. It has been tested to work
+with Guava 23.
+
+### Updated RFile to prevent very large blocks
+
+RFiles now use windowed statistics ([ACCUMULO-4669]) to prevent very large blocks.  In 1.8.0 a bug was introduced that
+caused RFile data block sizes to grow very large in the case where key sizes slowly increased.  This could lead
+to degraded query performance or out of memory exceptions on tablet servers.
 
 ## Other Notable Changes
 
+* [#403] - Enabled more metrics reporting
+* [#410] - Fixed inefficient auths check
+* [ACCUMULO-4528] - Add import/export table info to docs
+* [ACCUMULO-4636] - System iterator perfomance improvements
+* [ACCUMULO-4643] - Allow iterators to yield
```

Comment:
This is only other one I would think users might want to know.  Maybe @ivakegg can write up a snippet?